sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/SviCRUDTests.cs

```diff
@@ -0,0 +1,268 @@
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+            CleanupResourceGroups();
+        }
+
+        /// <summary>
+        /// The SVI CRUD test that creates and installs
+        /// an SVI.
+        /// </summary>
+        [TestCase]
+        [RecordedTest]
+        public async Task TestSVICrudOperations()
+        {
+            // Create Test RG
+            var resourceGroupName = Recording.GenerateAssetName("SdkRg-SVICrud");
+            SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+            ResourceGroupResource rg = await CreateResourceGroup(
+                subscription,
+                resourceGroupName,
+                AzureLocation.EastUS);
+
+            SapVirtualInstanceData SviData = this.GetSingleServerPayloadToPut(rg.Data.Name, false);
+
+            SapVirtualInstanceData SviInstallData = this.GetSingleServerPayloadToPut(rg.Data.Name, true);
+
+            // Create SVI
+            try
+            {
+                ArmOperation<SapVirtualInstanceResource> resource = await rg.GetSapVirtualInstances().CreateOrUpdateAsync(
+                    waitUntil: WaitUntil.Completed,
+                    sapVirtualInstanceName: "F95",
+                    data: SviData);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+            SapVirtualInstanceCollection collection = rg.GetSapVirtualInstances();
+
+            // Install SVI
+            try
+            {
+                ArmOperation<SapVirtualInstanceResource> resource = await rg.GetSapVirtualInstances().CreateOrUpdateAsync(
+                    waitUntil: WaitUntil.Completed,
+                    sapVirtualInstanceName: "F95",
+                    data: SviInstallData);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+            // Delete RG
+            collection = rg.GetSapVirtualInstances();
+            await rg.DeleteAsync(WaitUntil.Completed);
+        }
+
+        [TestCase]
+        [RecordedTest]
+        public async Task TestSVIStartStopOperations()
+        {
+            SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+            var lro = await subscription.GetResourceGroups().GetAsync("E2E-SVI-DevBox-16Jun-avzone-RHEL-SAP-HA-84sapha-gen2");
+            ResourceGroupResource rg = lro.Value;
+            var sviObject1 = await lro.Value.GetSapVirtualInstanceAsync("F62");
+
+            int count = 0;
+            var sviObjectstatus = lro.Value.GetSapVirtualInstance("A13").Value.Data.Status;
+            do
+            {
+                Thread.Sleep(GetOpsIntervalinMillis);
+                sviObjectstatus = lro.Value.GetSapVirtualInstance("A13").Value.Data.Status;
+                count++;
+            }
+            while (sviObjectstatus != SapVirtualInstanceStatus.Offline &&
+                count < 6);
```

Comment:
why are we waiting on a resource we just performed a get on? should'nt we call start/stop first?